nilearn/input_data/tests/test_nifti_masker.py

```diff
@@ -110,6 +111,37 @@ def test_mask_3d():
         assert_raises(TypeError, masker.fit)
 
 
+def test_mask_4d():
+    # Dummy mask
+    mask = np.zeros((10, 10, 10))
+    mask[3:7, 3:7, 3:7] = 1
+    mask_img = Nifti1Image(mask, np.eye(4))
+    n_mask_vox = len(np.where(mask == True)[0])
```

Comment:
This is quite convoluted. I am sure there must be something better than this. mask.sum() should work but maybe there is something more generic.